Editor/Scripts/Analytics/MeshSyncObserver.cs

```diff
@@ -0,0 +1,61 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+using UnityEditor;
+
+namespace Unity.MeshSync.Editor.Analytics {
+    
+
+    /// <summary>
+    /// Hacky way to restore observers to MeshSync servers in scene
+    /// </summary>
+    [InitializeOnLoad]
+    public static class MeshSyncObserverStartUp {
+
+        private static void update() {
+
+            var array = UnityEngine.Object.FindObjectsOfType<BaseMeshSync>();
+
+            foreach (var server in array) {
+                if (server.getNumObservers == 0) {
+
+                    var observer = new MeshSyncObserver();
```

Comment:
Shouldn't we subscribe in `MeshSyncServer.OnCreate()` or somewhere similar instead of doing this check every frame ?

Reply:
> This is to cover the scenario where the server is being deserialized.

I am not sure I understand.
What is the drawback if we do this subscribe in `MeshSyncServer.OnEnable()` for instance ?

Reply:
I don't think we can do that since the code that needs to run is in UnityEditor

Reply:
This was fixed by unsubscribing after successful first run, so this only runs once

Reply:
Just a comment. Discussion continues in another thread:
https://github.com/unity3d-jp/MeshSync/pull/778#discussion_r987964338